VirtualLibrary/Pages/Books/Edit.cshtml.cs

```diff
@@ -1,21 +1,18 @@
-﻿using System;
```

Comment:
![image](https://user-images.githubusercontent.com/3893234/192390159-b5452042-7cb1-407b-aadf-ad8ac9d862b9.png)

Access was denied while logging in as librarian seeded user

Reply:
And delete

Reply:
Create does not work

Reply:
A user can,
1. Check a book out
2. Read (list) what books are available

A librarian can,
1. Check a book back in
2. Read (list) what books are available
3. Edit a book
4. Delete a book
5. Create a book